Datasets/data-connect-dataset-teamsstandardchannelmessages.md

```diff
@@ -41,15 +41,16 @@ The TeamsStandardChannelMessages_v0 dataset can be joined to other Teams dataset
 | SentDateTime |	datetime |	UTC formatted sent date time of message. | Yes | Date |
 | Subject | string |	Subject of the message or announcement.	| No	| None |
 | Importance |	string |	The importance defined by the sender. |	No |	None |
-| From	| string |	Nested property outlining the sender of the message. |	No |	None |
-| Body |	string |	The message text body. |	No |	None |
+| From	| object |	Nested property outlining the sender of the message. |	No |	None |
+| Body |	object |	The message text body. |	No |	None |
 | ParentFolderId |	string |	The unique identifier for the teams message parent folder. |	No |	None |
 | ConversationId |	string |	The ID of the conversation the email belongs to. |	No |	None |
+| Mentions | array |All the @ mentions in body of the message.|`[{\"id\": 1024, \"mentionText\": \"test mention\", \"mentioned\": {\"@odata.type\": \"user\"}]`|0|false|
```

Comment:
Since this "mentions" field was not originally there in schema, but in the sample data I see 1 row entry where Mentions field is present, (though the json looks malformed), can you check:
1) If the dataset actually returns Mentions field?
2) If so, can you please correct the row of sample data so that it is not malformed
3) Incase this Mentions field is to be retained, please make last 2 columns as "No" and "None" instead of "0" and "false"

Reply:
i remember that the mentions field is not populated and there was a thread related to this long time ago. Added this here by mistake. The sample run is giving null response. It was confirmed from the api owners that the data is not populated by them. Removing this field in the latest commit